bitnami/fluentd/templates/svc-headless.yaml

```diff
@@ -6,9 +6,23 @@ metadata:
   labels: {{- include "fluentd.labels" . | nindent 4 }}
     app.kubernetes.io/component: aggregator
     app: aggregator
+  {{- if .Values.aggregator.service.annotations }}
```

Comment:
Does it make sense to keep considering this a headless service if it may have resource allocation? For instance, if you choose `service.type` `LoadBalancer`.

https://kubernetes.io/docs/concepts/services-networking/service/#headless-services

Reply:
Totally agree, that once this is no longer a headless service that the name should change. What's your take on it? 

Would you prefer to keep the headless service (to avoid any breaking change), but if a service of type LoadBalancer was defined, then a new service is deployed apart from the headless?

Reply:
> Would you prefer to keep the headless service (to avoid any breaking change), but if a service of type LoadBalancer was defined, then a new service is deployed apart from the headless?

That approach sounds great, @fsschmitt. Will you update the PR?

Reply:
Yes, I'll update this afternoon with this approach. Thanks!

Reply:
@andresbono, I've updated with the following:
- Reverted any change on the `svc-headless.yaml` file;
- Added (on the same fashion as the forwarder-svc) the `aggregator-svc.yaml` which enables the same level of configuration on a service of its own;